fdb-relational-grpc/src/main/proto/grpc/relational/jdbc/v1/result_set.proto

```diff
@@ -41,6 +42,10 @@ message ResultSet {
   // Each element in `rows` is a row whose format is defined by metadata. The ith element
   // in each row matches the ith field in ResultSetMetadata.
   repeated Struct row = 2;
+
+  // Optional continuation that can continue the query from the point the current results ended.
+  // Not all queries support continuations. Null (or non existent) continuations means "no continuation".
```

Comment:
What queries don't support continuations?

Reply:
That wasn’t accurate. The field is optional since some result sets do not support continuations - clarified.